app/metrics.yaml

```diff
@@ -669,7 +669,7 @@ activation:
     - https://github.com/mozilla-mobile/fenix/pull/1707#issuecomment-486972209
     notification_emails:
     - fenix-core@mozilla.com
-    expires: "2019-10-01"
+    expires: "2019-03-01"
```

Comment:
Do we also want to change the year here?

Reply:
lol yes omg